pkg/operator/targetconfigcontroller/targetconfigcontroller_test.go

```diff
@@ -88,14 +96,70 @@ func TestTargetConfigController(t *testing.T) {
 				u.FakeEtcdMemberWithoutServer(0),
 				u.FakeEtcdMemberWithoutServer(2),
 			},
+			etcdBackupSpec:    nil,
 			etcdMembersEnvVar: "1,3",
 		},
+		{
+			name: "BackupVar Test HappyPath",
+			objects: []runtime.Object{
+				u.BootstrapConfigMap(u.WithBootstrapStatus("complete")),
+			},
+			staticPodStatus: u.StaticPodOperatorStatus(
+				u.WithLatestRevision(3),
+				u.WithNodeStatusAtCurrentRevision(3),
+				u.WithNodeStatusAtCurrentRevision(3),
+				u.WithNodeStatusAtCurrentRevision(3),
+			),
+			etcdMembers: []*etcdserverpb.Member{
+				u.FakeEtcdMemberWithoutServer(0),
+				u.FakeEtcdMemberWithoutServer(1),
+				u.FakeEtcdMemberWithoutServer(2),
+			},
+			etcdBackupSpec:    u.CreateEtcdBackupSpecPtr("GMT", "0 */2 * * *"),
+			etcdMembersEnvVar: "1,2,3",
```

Comment:
I've just noticed and this is not related to your PR, but is this envvar even used?

Reply:
not at all, shall i remove ?

Reply:
I'll do another PR after this one merged, let's not further complicate :rofl:

Reply:
sgtm 👍🏽